mesmerize_napari/mcorr_viz_gui.py

```diff
@@ -0,0 +1,58 @@
+from PyQt5 import QtWidgets
```

Comment:
how does this differ from the mcorr ds_avg that you just did?

Reply:
It doesn't anymore. check recent push